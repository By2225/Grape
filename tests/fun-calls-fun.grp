fun Int apple() {
    return peach();
}
fun Int peach() {
    return 3;
}
fun Int main() {
    print(apple());
    return 0;
}
