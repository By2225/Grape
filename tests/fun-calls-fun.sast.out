
fun Int main() {
(* : print((Int : apple())));
return (Int : 0);
}

fun Int peach() {
return (Int : 3);
}

fun Int apple() {
return (Int : peach());
}
